src/ethereum/arrow_glacier/fork_types.py

```diff
@@ -14,7 +14,7 @@
 
 from dataclasses import dataclass
 from typing import Tuple, Union
-
+from .transactions import LegacyTransaction
```

Comment:
```suggestion
if TYPE_CHECKING:
    from .transactions import LegacyTransaction
```

Reply:
ok will do the above changes and then commit them and say thnks for the help @SamWilsn